neural_lam/datastore/mllam.py

```diff
@@ -0,0 +1,355 @@
+# Standard library
+import shutil
+import warnings
+from pathlib import Path
+from typing import List
+
+# Third-party
+import cartopy.crs as ccrs
+import mllam_data_prep as mdp
+import xarray as xr
+from loguru import logger
+from numpy import ndarray
+
+# Local
+from .base import BaseCartesianDatastore, CartesianGridShape
+
+
+class MLLAMDatastore(BaseCartesianDatastore):
```

Comment:
This name is a bit confusing, since all datastores are datastores for ML LAM. I guess the MLLAM here refers to `mllam_data_prep`, but I don't expect many to make this connection. This datastore is specifically for reading from a single `xarray.Dataarray`. So something like `DataArrayStore`, `SingleArrayDataStore` or `XarrayDatastore` could maybe be more descriptive.

Reply:
Yes, you're right the name is referring to `mllam_data_prep` python package and I agree, the name is confusing... Could we call `MDPDatastore` instead (short-name would be `mdp` in the command line). I was thinking it would be good to convey that the transformations are done in a different package, not that the current name really does that well, but that was my aim

Reply:
Yes, I think that works well. Agree that there is a point in being clear that transformations happen in mdp